books-call-number/nodes/query.json

```
@@ -0,0 +1,26 @@
+{
+  "id": "78ae1271-bcf8-452f-a3de-7e4583ba8017",
+  "name": "Query list of books by call number.",
```

```suggestion
  "name": "Query by Call Number",
```
These are labels/titles and so should not have periods at the end of the sentence.

---

Resolved